app.js

```diff
@@ -38,113 +38,111 @@ var Storage = multer.diskStorage({
     }
 });
 var upload = multer({
-    storage: Storage
+    storage: storage
 }).array("images", 3);
 app.get('/login' , function (req , res) {
-    if(req.session.user) {
+    if(req.session.username) {
         res.redirect('/admin');
     }else {
         res.sendFile(__dirname + '/www/login.html');
     }
 });
 app.post('/login' , function (req , res) {
-    login(req , res);
+    const post = req.body;
+    const username  = sanitizeHtml(post.user);
+    const password = sanitizeHtml(post.password);
+    var sql = "SELECT * FROM login WHERE username='"+username+"'";
+    con.query(sql, function (err, result, fields) {
+        if( result.length > 0 && result[0].password === password ) {
+            console.log("Auth set");
+            req.session.username = post.user;
+            res.send({"result" : "Found"});
+        }else {
+            res.send({"result": "NotFound"});
+        }
+    });
 });
 app.get('/admin'  ,  function (req , res) {
-    console.log(req.session.user);
-    if(!req.session.user) {
-        console.log("Admin");
+    if(!req.session.username) {
         res.redirect('/login');
     }else {
         res.sendFile(__dirname + '/www/admin.html');
     }
 });
-app.get('/speaker' , function (req , res) { 
+app.get('/speakers' , function (req , res) {
 	res.sendFile(__dirname + '/www/speaker.html');
 });
-app.post('/speaker' , function (req , res) { 
+app.post('/speakers' , function (req , res) {
```

Comment:
This should be get method, since we are getting list of speakers.

Let's not merge API and static pages, we can put all api requests under /api/* route.